collections/_article/year-in-review-2024.md

```diff
@@ -0,0 +1,445 @@
+---
+title: "2024 Cherry-picks"
+excerpt: "This year was almost as packed as the 4.3 release, so let's go through the months together and talk about a few memorable moments as well as tech highlights by engine area."
```

Comment:
```suggestion
excerpt: "Last year was almost as packed as the 4.3 release, so let's go through the months together and talk about a few memorable moments as well as tech highlights by engine area."
```
or
```suggestion
excerpt: "2024 was almost as packed as the 4.3 release, so let's go through the months together and talk about a few memorable moments as well as tech highlights by engine area."
```
Having said that, that first part of the sentence ("(year) was almost as packed as the 4.3 release") reads a bit oddly!